src/main/kotlin/com/sourcegraph/cody/agent/CodyAgent.kt

```diff
@@ -161,7 +161,10 @@ private constructor(
               .redirectErrorStream(false)
               .redirectError(ProcessBuilder.Redirect.PIPE)
               .start()
-      process.onExit().thenAccept { token.abort() }
+      process.onExit().thenAccept {
+        logger.warn("Cody agent process exited with code " + it.exitValue())
```

Comment:
Do we want to print that for 0 code as well? Then I think it just add noise to the log

Reply:
Sure, I'lll fix that.